tox.ini

```diff
@@ -23,7 +22,8 @@ commands =
     make test_quality
 
 [testenv:docs]
-deps = 
+deps =
+    setuptools
```

Comment:
This shouldn't be needed for Python 3.11 support as `setuptools` is removed in `Python 3.12`.